pkg/executor/build.go

```diff
@@ -160,30 +161,37 @@ func (s *stageBuilder) build() error {
 			return err
 		}
 		files = command.FilesToSnapshot()
-		var contents []byte
 
 		if !s.shouldTakeSnapshot(index, files) {
 			continue
 		}
 
+		f, err := ioutil.TempFile("/kaniko", "")
```

Comment:
is there any case where we would want this to be in-memory instead? (i.e. configurable)